### ISSUE TITLE
Multiple OIDC providers

### ISSUE THREAD

**Issue by Saganic** [timalanfarrow/kratos-selfservice-ui-vue3-typescript#7]
When using multiple OIDC providers for Login/Registration, the method value is taken as the first value passed through. This means no matter which provider you click, you will get the first one.

EDIT: Changing the FormButton component to an input element results in correct behaviour.

EDIT: Also appears to work if you make the id unique:
```
template
  .button(:class='variant')
    label.open-sans.caps.fs-2.fw_6(:for='`${id}-${value}`')
      | {{ label }}
    button(:id='`${id}-${value}`', :type='type', :name='name', :value='value')
```
in FormButton.vue

I would open a PR, but we don't work with Pug so my codebase is pretty different to yours! 🦆

**Comment by timalanfarrow** [comment]
@Saganic Thank you for the update, I'll get this fixed.

**Comment by timalanfarrow** [comment]
Fixed in #9
